File: JavaScript_Courses/JavaScript_Basics/Preparation/03_Conditional_Statements_Advanced_Exercise/05_Journey.js
function journey(input) {
    // •	Втори–  Един от двата възможни сезона: "summer" или "winter"
    let budget = Number(input[0]);
    let season = input[1];

    let destination;
    let vacationType;
    let costAsPercent = 0;

    if (budget <= 100) {
        destination = "Bulgaria";
        if (season == "summer") {
            vacationType = "Camp";
            costAsPercent = 0.30;
        } else if (season == "winter") {
            vacationType = "Hotel";
            costAsPercent = 0.70;
        }
    } else if (budget <= 1000) {
        destination = "Balkans";
        if (season == "summer") {
            vacationType = "Camp";
            costAsPercent = 0.40;
        } else if (season == "winter") {
            vacationType = "Hotel";
            costAsPercent = 0.80;
        }
    } else {
        destination = "Europe";
        vacationType = "Hotel";
        costAsPercent = 0.90;
    }

    console.log(`Somewhere in ${destination}`);
    console.log(`${vacationType} - ${(budget * costAsPercent).toFixed(2)}`);
}

// journey(["75", "winter"]);

File: Python_Courses/Python_Basics/Python_PB_2025_03/03_Conditional_Statements_Advanced_Exercise/05_Journey.py
budget = float(input())
season = input()  # "summer” or "winter”

destination = ''
vacation_type = ''
total_cost = 0.0

if budget <= 100:
    destination = 'Bulgaria'
    if season == 'summer':
        total_cost = budget * 0.30  # Лято - 30% от бюджета
        vacation_type = 'Camp'
    elif season == 'winter':
        total_cost = budget * 0.70 # Зима - 70% от бюджета
        vacation_type = 'Hotel'

elif budget <= 1_000:  # (100; 1000]
    destination = 'Balkans'
    if season == 'summer':
        total_cost = budget * 0.40  # Лято - 40% от бюджета
        vacation_type = 'Camp'
    elif season == 'winter':
        total_cost = budget * 0.80  # Зима - 80% от бюджета
        vacation_type = 'Hotel'
else:  # (1000; +inf)
    destination = 'Europe'
    total_cost = budget * 0.90  # независимо от сезона, ще похарчи 90% от бюджета
    vacation_type = 'Hotel'  # aко е в Европа, независимо от сезона, ще почива в хотел
    # if season == 'summer':
    #     total_cost = budget * 0.90  # независимо от сезона, ще похарчи 90% от бюджета
    #     vacation_type = 'Hotel'
    # elif season == 'winter':
    #     total_cost = budget * 0.90  # независимо от сезона, ще похарчи 90% от бюджета
    #     vacation_type = 'Hotel'

print(f'Somewhere in {destination}')
print(f'{vacation_type} - {total_cost:.2f}')
